expressions/table_generator/GeneratorFunction.hpp

```diff
@@ -0,0 +1,119 @@
+/**
+ *   Copyright 2016 Pivotal Software, Inc.
```

Comment:
Lint complains when the copyright notice is dropped ...
`./expressions/table_generator/GenerateSeries.hpp:0:  No copyright message found.  You should have a line: "Copyright [year] <Copyright Owner>"  [legal/copyright] [5]`